<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-QJLSV9M" failures="3" tests="5" name="RESTAPITest" time="9.077" errors="0" timestamp="2023-05-06T13:08:35 IST">
  <testcase classname="com.w2a.zoho.utilities.FactoApi" name="LoginjwtToken" time="3.124"/>
  <testcase classname="com.w2a.zoho.utilities.FactoApi" name="myTest" time="0.61"/>
  <testcase classname="com.w2a.zoho.utilities.FactoApi" name="myTest" time="0.498">
    <failure type="java.lang.AssertionError" message="expected [200] but found [502]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [502]
at com.w2a.zoho.utilities.FactoApi.myTest(FactoApi.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
  <testcase classname="com.w2a.zoho.utilities.FactoApi" name="myTest" time="1.915">
    <failure type="java.lang.AssertionError" message="expected [200] but found [502]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [502]
at com.w2a.zoho.utilities.FactoApi.myTest(FactoApi.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
  <testcase classname="com.w2a.zoho.utilities.FactoApi" name="myTest" time="1.572">
    <failure type="java.lang.AssertionError" message="expected [200] but found [502]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [502]
at com.w2a.zoho.utilities.FactoApi.myTest(FactoApi.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
</testsuite> <!-- RESTAPITest -->
